### PR TITLE
Changing the file

### PR DESCRIPTION
**Pull request by EkaterinaV886**
Hi, please check

### CODE REVIEW

**Review by AlenaGarnovskaya — APPROVED** [review]
This one works good'

File: 02_Front_end_testing/NasaAPIs.py [hunk 1]
```diff
@@ -0,0 +1,566 @@
...
+class FirefoxPositiveTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self.driver = webdriver.Firefox()
+
+        # Open the site page
+
+    def test_nasa_firefox(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

File: 02_Front_end_testing/NasaAPIs.py [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+        driver.find_element(By.XPATH, "//h3[contains(text(),'Querying by Date(s)')]").click()
+        time.sleep(1)
+        driver.find_element(By.XPATH, "(//h3[contains(@id,'example-query')])[8]").click()
+        time.sleep(1)
+
+        # Open the site page
+
+    def test_nasa_firefox1(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

File: 02_Front_end_testing/NasaAPIs.py [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+        driver.find_element(By.XPATH, "//h2[@id='exoPlanetIntro']").click()
+        time.sleep(1)
+        driver.find_element(By.XPATH, "//h2[@id='exoPlanetExamples']").click()
+        time.sleep(1)
+
+        # Open the site page
+
+    def test_nasa_firefox2(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

File: 02_Front_end_testing/NasaAPIs.py [hunk 4]
```diff
@@ -0,0 +1,566 @@
...
+        driver.find_element(By.XPATH, "//h2[contains(.,'API Requests with Python')]").click()
+        time.sleep(1)
+        driver.find_element(By.XPATH, "//h2[contains(.,'Resources')]").click()
+        time.sleep(1)
+
+        # Open the site page
+
+    def test_nasa_firefox3(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

File: 02_Front_end_testing/NasaAPIs.py [hunk 5]
```diff
@@ -0,0 +1,566 @@
...
+        driver.find_element(By.XPATH, "(//h3[contains(@id,'query-parameters')])[6]").click()
+        time.sleep(1)
+        driver.find_element(By.XPATH, "//img[@src='./assets/insight/insight_mars_wind_rose.png']").click()
+        time.sleep(1)
+
+        # Open the site page
+
+    def test_nasa_firefox4(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

File: 02_Front_end_testing/NasaAPIs.py [hunk 6]
```diff
@@ -0,0 +1,566 @@
...
+        time.sleep(1)
+        driver.find_element(By.XPATH, "//h3[contains(.,'Mission Manifest')]").click()
+        time.sleep(1)
+
+        # Open the site page
+
+    def test_nasa_firefox5(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
+        driver.get("https://api.nasa.gov/")
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

File: 02_Front_end_testing/NasaAPIs.py [hunk 7]
```diff
@@ -0,0 +1,566 @@
...
+        driver.find_element(By.XPATH, "//h3[@id='tle-http-request']").click()
+        time.sleep(1)
+        driver.find_element(By.XPATH, "(//h3[contains(@id,'example-query')])[15]").click()
+        time.sleep(1)
+
+        # Open the site page
+
+    def test_nasa_firefox6(self):
+        driver = self.driver
+        driver = webdriver.Firefox()
```
Comment (sergei-orloff) [inline 02_Front_end_testing/NasaAPIs.py]: Remove
>>  driver = webdriver.Firefox()

**Review by sergei-orloff — APPROVED** [review]
Approved with changes:

Remove
>>  driver = webdriver.Firefox()

**Review by sergei-orloff — APPROVED** [review]
👍

**Review by anastasiia-mirus — APPROVED** [review]
Works great!

**Pull request merged**